actions/webhook.go

```diff
@@ -0,0 +1,94 @@
+package actions
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+	"time"
+)
+
+type Webhook struct {
+	ID         string
+	ActionName string
+	URL        string
+}
+
+type WebhookEventInfo struct {
```

Comment:
I don't like that our API with customers is being handled this way. It's so easy to break it without ever knowing.

Reply:
For now, I'll keep it as is, until we will have to address compatibility changes and need to address versioning.